src/client.cpp

```diff
@@ -23,30 +23,33 @@ using namespace ADDON;
 std::string      g_szHostname             = DEFAULT_HOST;                  ///< The Host name or IP of the NextPVR server
 std::string      g_szPin                  = DEFAULT_PIN;                   ///< The PIN for the NextPVR server
 int              g_iPort                  = DEFAULT_PORT;                  ///< The web listening port (default: 8866)
-int16_t          g_timeShiftBufferSeconds = 0;
+int g_timeShiftBufferSeconds = 0;
 std::string      g_host_mac = "";
 eStreamingMethod g_livestreamingmethod = RealTime;
 eNowPlaying      g_NowPlaying = NotPlaying;
 int              g_wol_timeout;
 bool             g_wol_enabled;
 bool             g_KodiLook;
-bool             g_eraseIcons = false;
-int              g_iResolution;
 
 /* Client member variables */
 ADDON_STATUS           m_CurStatus    = ADDON_STATUS_UNKNOWN;
 cPVRClientNextPVR     *g_client       = NULL;
 std::string            g_szUserPath   = "";
 std::string            g_szClientPath = "";
-bool             g_bUseTimeshift;  /* obsolete but settings.xml might have it */
 
 CHelper_libXBMC_addon *XBMC           = NULL;
 CHelper_libXBMC_pvr   *PVR            = NULL;
 bool                   g_bDownloadGuideArtwork = false;
 
+int g_backendVersion{ 0 };
```

Comment:
Just a comment but you could consider introducing a settings class in the future instead of using global variables.

Reply:
Already done as noted

Reply:
Already done